RedHat-UX/red-hat-design-system
# docs: redo Color page

## What I did

1.


## Testing Instructions

1. Check [DP](https://deploy-preview-1321--red-hat-design-system.netlify.app/foundations/color/)

## Notes to Reviewers

@coreyvickery, some changes while you were out:
- On the "Theming" subpage, I changed "Custom colors" to "Custom theme" because Ryan said Brand will reject requests for new colors. I figured custom theme is what you meant.
- I updated the content under "Usage" > "Surface". The background of a button isn't really considered a surface, so the paragraph had to be reworded a bit.
- I also replaced "component" with "element and pattern" wherever it appear, except for the "Graphical objects and UI components" section.
